hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/SupportTeamServiceProperties.java

```diff
@@ -0,0 +1,97 @@
+package pl.allegro.tech.hermes.management.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.net.URL;
+import java.util.Optional;
+
+@ConfigurationProperties(prefix = "supportTeam")
+public class SupportTeamServiceProperties {
+
+    private SupportTeamServiceType type;
+
+    private CrowdProperties crowd;
+
+    private Long cachingTime = 300L;
```

Comment:
millis? seconds? i would set sth like `cacheDurationSeconds` - it is always easier to read stuff like this